[2024-06-28T23:59:44.400+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T23:59:44.412+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_transfer_to_gcs.export_data_from_mysql manual__2024-06-28T23:59:42.915839+00:00 [queued]>
[2024-06-28T23:59:44.418+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_transfer_to_gcs.export_data_from_mysql manual__2024-06-28T23:59:42.915839+00:00 [queued]>
[2024-06-28T23:59:44.419+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T23:59:44.428+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): export_data_from_mysql> on 2024-06-28 23:59:42.915839+00:00
[2024-06-28T23:59:44.435+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=447) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-28T23:59:44.436+0000] {standard_task_runner.py:63} INFO - Started process 449 to run task
[2024-06-28T23:59:44.434+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_transfer_to_gcs', 'export_data_from_mysql', 'manual__2024-06-28T23:59:42.915839+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/to_gcs.py', '--cfg-path', '/tmp/tmpb7y28nva']
[2024-06-28T23:59:44.436+0000] {standard_task_runner.py:91} INFO - Job 158: Subtask export_data_from_mysql
[2024-06-28T23:59:44.475+0000] {task_command.py:426} INFO - Running <TaskInstance: data_transfer_to_gcs.export_data_from_mysql manual__2024-06-28T23:59:42.915839+00:00 [running]> on host 1574d6467ca9
[2024-06-28T23:59:44.552+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_transfer_to_gcs' AIRFLOW_CTX_TASK_ID='export_data_from_mysql' AIRFLOW_CTX_EXECUTION_DATE='2024-06-28T23:59:42.915839+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-28T23:59:42.915839+00:00'
[2024-06-28T23:59:44.553+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T23:59:44.575+0000] {base.py:84} INFO - Using connection ID 'mysql_conn' for task execution.
[2024-06-28T23:59:44.941+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/datasource/datasource.csv
[2024-06-28T23:59:44.942+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T23:59:44.960+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_transfer_to_gcs, task_id=export_data_from_mysql, run_id=manual__2024-06-28T23:59:42.915839+00:00, execution_date=20240628T235942, start_date=20240628T235944, end_date=20240628T235944
[2024-06-28T23:59:45.010+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T23:59:45.038+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T23:59:45.040+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
